modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/AbstractModelLoaderDelegate.java

```diff
@@ -43,6 +47,15 @@
     this.loader = loader;
   }
 
+  protected ClassLoader getExtensionClassLoader() {
+    Class<?> extensionType = getExtensionType();
+    if (extensionType != null) {
+      return extensionType.getClassLoader();
+    } else {
+      return ExtensionModel.class.getClassLoader();
```

Comment:
TCCL

Reply:
I Can't 😞 , If I do that I will get the Maven Plugin Classloader which doesn't have visibility to some Mule Classes.

Reply:
Override this method on each implementation, and by default in the abstract return the TCC

Reply:
Implementations of this class are: `OperationModelLoaderDelegate`, `ParameterModelLoaderDelegate`, etc, which doesn't make sense that each one of this have to redefine these